packages/wallet/src/ui/Form.tsx

```diff
@@ -63,7 +63,7 @@ export const Form = <T extends FieldValues>({
 }: FormProps<T>) => {
   return (
     <FormProvider {...form}>
-      <form onSubmit={form.handleSubmit(onSubmit)} {...props}>
+      <form onSubmit={form.handleSubmit(onSubmit)} {...props} className="w-3/5">
```

Comment:
Let's not apply styles to the form abstraction. We can just create a wrapper around the `Form` component and it will have it's parent width.